include/aspect/particle/property/crystal_preferred_orientation.h

```diff
@@ -222,6 +240,39 @@ namespace aspect
           unsigned int
           get_number_of_minerals();
 
+          /**
+           * @brief Deterimens the deformation type from the deformation type selector.
+           *
+           * @param stress
```

Comment:
This documentation is incomplete and empty. Either complete it or remove it, otherwise doxygen will warn.